storm-core/src/native/worker-launcher/impl/worker-launcher.c

```diff
@@ -485,14 +482,14 @@ int setup_stormdist_dir(const char* local_dir) {
 
       case FTS_DP:        // A directory being visited in post-order
       case FTS_DOT:       // A dot directory
+      case FTS_SL:        // A symbolic link
+      case FTS_SLNONE:    // A broken symbolic link
```

Comment:
I'm also not sure why we want the symlink/a broken symlink to be a noop?  We use lchown to not follow the symlink, so we will be sure to change ownership of the symlink itself (otherwise it will still be owned by the supervisor user).  Was chmod causing issues?  If so we probably want to split the chown and chmod portions and only call chmod on things that are not symlinks.